tests/test_delete_all_links.py

```diff
@@ -2,13 +2,9 @@
 from requests import request
 
 
-def test_delete_all_links_status_code():
+def test_delete_all_links_status_code(create_shortcut_link):
     # precondition
-    request(
-        method='POST',
-        url='http://localhost:8888/shortcut',
-        data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
-    )
+    create_shortcut_link
```

Comment:
Wtf? I think you could delete this "#precondition" section.

Reply:
Why? We create some shortcut to be sure that we really delete something with delete request